### PR TITLE
5 ways we work as a team at Mixlr

### PR DESCRIPTION
**Pull request by rfwatson**
Anybody have 5 minutes to proof-read?

### CODE REVIEW

**Comment by rfwatson** [comment]
@platzhirsch @epscylonb

**Comment by epscylonb** [comment]
Nicely written, didn't spot any glaring mistakes.

File: _posts/2016-03-05-ways-we-work-as-a-team-at-mixlr.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+comments: true
+tags: process, agile, tools, team
+---
+
+<a name="top"></a>
+<p class="info_block"><strong>We're hiring.</strong> If you enjoy reading this post, you'll find Mixlr is a great team to be a part of. <a href="/jobs">Visit the Mixlr jobs homepage</a>.</p>
+
+At Mixlr, we’ve racked up more than five years of experience working as a team to create our service.
+
+Along the way, we’ve also accumulated a few methods that help us work together effectively as a team on a day-to-day basis. Here’s some of our favourites.
```
Comment (konradreiche) [inline _posts/2016-03-05-ways-we-work-as-a-team-at-mixlr.md]: Here's some or Here're some?

**Comment by konradreiche** [comment]
Great post, looking forward to see it live :smiley: :memo:

**Pull request merged**